pkgs/development/libraries/enchant/2.x.nix

```diff
@@ -13,13 +13,13 @@
 
 stdenv.mkDerivation rec {
   pname = "enchant";
-  version = "2.6.9";
+  version = "2.7.1";
 
   outputs = [ "out" "dev" ];
 
   src = fetchurl {
```

Comment:
Since the project switched to Vala and the tarballs now contain ton of generated C, I wonder if we should continue using the tarballs from supply chain security perspective.